client/src/components/Logout/Logout.js

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from "react";
+import AuthService from "../../Services/AuthService";
+// import { AuthContext } from "../Context/AuthContext";
+
+
+
+function Logout(props) {
+  // const { isAuthenticated, user, setIsAuthenticated, setUser } = useContext(AuthContext);
+
+  const logoutHandler = () => {
+    AuthService.logout().then((data) => {
+      console.log(data);
```

Comment:
Do we want to be console logging the data from the logout method?

Reply:
removed